releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -0,0 +1,9 @@
+## Azure App Configuration Push Extension release notes
+The Azure App Configuration Push extension in Azure DevOps pipeline has been published to
+[Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push&ssr=false#overview).
+The version of the Azure App Configuration task being used will print to the console when the task is executed:

```

Comment:
Don't start a new line.

"The version of the Azure App Configuration *Push pipeline* task ~~being used~~ will print to the console when the task is executed:"